dev/service-registry/registration-javadsl/src/main/scala/com/lightbend/lagom/internal/server/ServiceRegistrationModule.scala

```diff
@@ -49,20 +46,20 @@ object ServiceRegistrationModule {
    */
   @Singleton
   private class RegisterWithServiceRegistry @Inject() (
```

Comment:
Maybe this class (and its inconsistently-named Scala API counterpart, `ServiceRegistration`) are candidates for refactoring to use the new `ServiceRegistryClient` abstraction introduced in #1477.

Reply:
Raised as #1489